shared/utils/gov_utils.go

```diff
@@ -0,0 +1,75 @@
+package utils
+
+import (
+	"reflect"
+	"strings"
+
+	"github.com/pokt-network/pocket/runtime/genesis"
+)
+
+// init initializes a map that contains the metadata extracted from `gov.proto`.
+// Since protobuf files do not change at runtime, it seems efficient to do it here.
+func init() {
+	GovParamMetadataMap = parseGovProto()
```

Comment:
Can we verify that `GovParams` (the map you have hardcoded elsewhere) corresponds to `gov.proto`?

I basically want a compile check to make sure we don't edit one file and forget to update the source code.

Reply:
I am unsure what you mean? I have removed the other maps that do that same thing to consolidate them here. This reflects the `genesis.Params` struct and doesn't need any updating if that changes.

Reply:
Leaving a comment in `utility/unit_of_work/gov.go`